### PR TITLE
adding post thanking Greg

### PR DESCRIPTION
**Pull request by tracykteal**

### CODE REVIEW

**Comment by tracykteal** [comment]
@ethanwhite @hlapp @karthik @kcranston @apawlik please review and PR if you think OK and appropriate timing. Greg's post just went up.

File: _posts/2016-12-14-thanks-to-greg.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
...en lesson development, of teaching shared curriculum, of learning to be a better teacher and most of all he created a community of people who are excited to work together. He made those of us who felt alone in our labs working in programming or data analysis feel like we weren’t alone and that we were connected to other people who shared our enthusiasm for data and code and our passion to share that perspective with others. When we did a survey of instructors, the most common reason people said they wanted to become instructors is because they talked to Greg. Greg has collectively and individually inspired over 700 instructors that we have today as well as countless learners in any of the workshops that he’s taught. He’s made us feel like we can do more than we thought and that we had something important to contribute. For Data Carpentry, he is the one who encouraged us to scale it beyond an initial few workshops and make it a Carpentry and was on the Steering Committee for the crucial time of getting it off the ground. We’re sure there are so many other stories from people who Greg inspired to be bold and create something new and go after something that was valuable but difficult.
```
Comment (wking) [inline _posts/2016-12-14-thanks-to-greg.md]: “enthusiastic” → “are enthusiastic”?

File: _posts/2016-12-14-thanks-to-greg.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
```
Comment (hlapp) [inline _posts/2016-12-14-thanks-to-greg.md]: I'd suggest we start not assuming that everyone know's perfectly well who _Greg_ is and what's happening with him. For example: "_As many of you will know by now, Greg Wilson, Director of Instructor Training for Software Carpentry (and thus indirectly also for Data Carpentry), has announced that he will step down from his position at the end of January 2017 to join Shopify as their Computer Science Education Lead. We're very excited ..."_

Comment (hlapp) [inline _posts/2016-12-14-thanks-to-greg.md]: Insert a paragraph break before _There aren’t words to express_.

**Comment by tracykteal** [comment]
Thanks for the updates @hlapp

**Pull request merged**